Scheduled long running test failed - Run ID: 13773700990

## Bug information 

This issue is automatically generated if the scheduled long running test fails. The Radius long running test operates on a schedule of every 2 hours everyday. It's important to understand that the test may fail due to workflow infrastructure issues, like network problems, rather than the flakiness of the test itself. For the further investigation, please visit [here](https://github.com/radius-project/radius/actions/runs/13773700990).

[AB#14917](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/14917)

```
{
    cli.go:341: [rad]       "code": "ResourceDeploymentFailure",
    cli.go:341: [rad]       "message": "Failed",
    cli.go:341: [rad]       "target": "/planes/radius/local/resourceGroups/kind-radius/providers/Applications.Core/extenders/pgs-resources-terraform-pgsapp",
    cli.go:341: [rad]       "details": [
    cli.go:341: [rad]         {
    cli.go:341: [rad]           "code": "RecipeDeploymentFailed",
    cli.go:341: [rad]           "message": "terraform apply failure: exit status 1\n\nError: Error acquiring the state lock\n\nError message: the state is already locked by another terraform client\nLock Info:\n  ID:        8af57078-c0e8-cfe3-be41-284935ea3b9b\n  Path:      \n  Operation: OperationTypeApply\n  Who:       rpuser@applications-rp-5b45d5d5d9-zj4q7\n  Version:   1.11.1\n  Created:   2025-03-10 06:12:31.199172278 +0000 UTC\n  Info:      \n\n\nTerraform acquires a state lock to protect the state from being written\nby multiple users at the same time. Please resolve the issue above and try\nagain. For most commands, you can disable locking with the \"-lock=false\"\nflag, but this is not recommended.\n"
    cli.go:341: [rad]         }
    cli.go:341: [rad]       ]
    cli.go:341: [rad]     }
    cli.go:341: [rad]   ]
    cli.go:341: [rad] }
    cli.go:341: [rad] 
    cli.go:341: [rad] TraceId:  2b75d6057ea69f9b7155e066d309137c
    cli.go:341: [rad] 
    cli.go:341: [rad] 
    deployexecutor.go:83: 
        	Error Trace:	/home/runner/work/radius/radius/test/step/deployexecutor.go:83
        	            				/home/runner/work/radius/radius/test/rp/rptest.go:392
        	Error:      	Received unexpected error:
        	            	code DeploymentFailed: err At least one resource deployment operation failed. Please see the details for the specific operation that failed.
        	Test:       	Test_TerraformRecipe_KubernetesPostgres/deploy_testdata/corerp-resources-terraform-postgres.bicep
        	Messages:   	failed to deploy deploy testdata/corerp-resources-terraform-postgres.bicep
    --- FAIL: Test_TerraformRecipe_KubernetesPostgres/deploy_testdata/corerp-resources-terraform-postgres.bicep (18.83s)

=== FAIL: test/functional-portable/corerp/noncloud/resources Test_TerraformRecipe_KubernetesPostgres (19.11s)
```

--- a/test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go
+++ b/test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go
@@ -143,7 +143,7 @@ func Test_TerraformRecipe_KubernetesPostgres(t *testing.T) {
 	template := "testdata/corerp-resources-terraform-postgres.bicep"
 	appName := "corerp-resources-terraform-pg-app"
 	envName := "corerp-resources-terraform-pg-env"
-	extenderName := "pgs-resources-terraform-pgsapp"
+	extenderName := "corerp-resources-terraform-pg"
 	secretName := "pgs-secretstore"
 	secretResourceName := appName + "/" + secretName
 	userName := "postgres"

--- a/test/functional-portable/corerp/noncloud/resources/testdata/corerp-resources-terraform-postgres.bicep
+++ b/test/functional-portable/corerp/noncloud/resources/testdata/corerp-resources-terraform-postgres.bicep
@@ -74,7 +74,7 @@ resource app 'Applications.Core/applications@2023-10-01-preview' = {
 }
 
 resource pgsapp 'Applications.Core/extenders@2023-10-01-preview' = {
-  name: 'pgs-resources-terraform-pgsapp'
+  name: 'corerp-resources-terraform-pg'
   properties: {
     application: app.id
     environment: env.id

